src/main/java/frc/robot/subsystems/drivetrain/commands/DriveForward.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.OI;
+import frc.robot.Robot;
+import frc.robot.Utils;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import frc.robot.valuetuner.WebConstant;
+import org.techfire225.webapp.FireLog;
+
+public class DriveForward extends CommandBase {
+
+    private SwerveDrive swerveDrive;
+    private WebConstant target = new WebConstant("targetSpeed", 0);
+
+    public DriveForward(SwerveDrive swerveDrive) {
+        this.swerveDrive = swerveDrive;
+        addRequirements(swerveDrive);
+    }
+
+    @Override
+    public void execute() {
+        double forward = Utils.joystickDeadband(-OI.getJoystickY());
+
+        for (int i = 0; i< 4; i++) {
+            swerveDrive.getModule(i).setAngle(swerveDrive.swerveModules[i].getAngle());
```

Comment:
I might be misunderstanding here something, but shouldn't all modules be pointed in the same direction?
Also, if you doing `set module angle to module angle`, you can just remove this line, as it does nothing.

Reply:
Ideally they should be, but at high speeds the wheels may drift and without this stablizing them there could be a problem.